src/quote/header.rs

```diff
@@ -0,0 +1,20 @@
+// SPDX-License-Identifier: Apache-2.0
+
+/// The type of attestation key used to sign the Report.
+#[derive(Debug, Clone, Copy, Eq, PartialEq)]
+#[non_exhaustive]
+#[repr(u16)]
+pub enum KeyType {
+    /// ECDSA-256-with-P-256 curve
+    ECDSA256P256 = 2,
+    /// ECDSA-384-with-P-384 curve; not supported
+    ECDSA384P384 = 3,
+}
```

Comment:
If this is following some external standard, can we link to the standard in a comment?

Reply:
Yeah I can add links to that

Reply:
I added a link for ECDSA but I can add a link to the entire structure in another PR